tests/test_detect_target_worker.py

```diff
@@ -15,7 +15,7 @@
 # from modules import points_and_time
 
 
-MODEL_PATH = "tests/model_example/yolov8s.pt"
+MODEL_PATH = "tests/model_example/yolov8s_pretrained_default.pt"
```

Comment:
Do you plan on creating a PR with the landing pad model?

Reply:
No, there are multiple models and they're pretty large. I was considering not including this one either but then the tests won't run properly without some extra setup.